ui/imports/shared/controls/AssetsDetailsHeader.qml

```diff
@@ -0,0 +1,107 @@
+import QtQuick 2.13
+import QtQuick.Layouts 1.13
+
+import utils 1.0
+import shared.controls 1.0
+
+import StatusQ.Components 0.1
+import StatusQ.Core.Theme 0.1
+import StatusQ.Core 0.1
+import StatusQ.Controls 0.1
+
+Item {
+    id: root
+
+    signal goBack()
+
+    property alias primaryText: tokenName.text
+    property alias secondaryText: cryptoBalance.text
+    property alias tertiaryText: fiatBalance.text
+    property var balances
+    property StatusImageSettings image: StatusImageSettings {
+        width: 40
+        height: 40
+    }
+    property var getNetworkColor: function(chainId){}
+    property var getNetworkIcon: function(chainId){}
+
+    implicitHeight: childrenRect.height
+
+    StatusFlatButton {
```

Comment:
`StatusBackButton` is the new cool thing :)

Reply:
<img width="385" alt="image" src="https://user-images.githubusercontent.com/60327365/184406205-0706c5f1-eaca-4bc3-a0c1-fcb3bb8243b9.png">
 the StatusBackButton unfortunately looks like this 
 
 what I need is as below 

<img width="385" alt="image" src="https://user-images.githubusercontent.com/60327365/184406351-ef83a56f-ac25-4426-a031-3b51b78bbd9b.png">

so I cant use it :(